browserpass.go

```diff
@@ -33,10 +33,12 @@ var endianness = binary.LittleEndian
 // The browser extension will look up settings in its localstorage and find
 // which options have been selected by the user, and put them in a JSON object
 // which is then passed along with the command over the native messaging api.
+
+// Config defines the root config structure sent from the browser extension
 type Config struct {
 	// Manual searches use FuzzySearch if true, GlobSearch otherwise
-	UseFuzzy bool     `json:"use_fuzzy_search"`
-	Paths    []string `json:"paths"`
+	UseFuzzy    bool                   `json:"use_fuzzy_search"`
+	CustomStore []pass.StoreDefinition `json:"customStore"`
```

Comment:
should we maybe call it in plural?

Reply:
Could you clarify what you mean by _"call it in plural"_ please? I'm not sure what you are asking me to do.

Reply:
sorry, I meant name it "CustomStores" 🙂

Reply:
Thanks for clarifying - done :-).